controllers/auth_controller.py

```
@@ -0,0 +1,53 @@
+    client_kwargs={
+        'scope': 'openid email profile https://www.googleapis.com/auth/gmail.readonly'
+    }
+)
+
+@router.get('/google')
+async def login_via_google(request: Request):
+    redirect_uri = request.url_for('auth_google_callback')
+    return await oauth.google.authorize_redirect(request, redirect_uri)
+
+@router.get('/google/callback')
+async def auth_google_callback(request: Request, db: Session = Depends(get_db)):
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)